components/brave_rewards/resources/shared/components/wallet_card/external_wallet_bubble.tsx

```diff
@@ -96,7 +96,7 @@ export function ExternalWalletBubble (props: Props) {
           externalWallet.status === 'pending' &&
             <style.pendingNotice>
               <PendingIcon />
-              <span>{getString('walletCompleteVerificationText')}</span>
+              <span>{formatMessage(getString('walletCompleteVerificationText'), [providerName])}</span>
```

Comment:
Total nit: would you mind adding some vertical whitespace to keep the line lengths <= 90ish?